### ISSUE TITLE
?-help-button aria-expanded value

### ISSUE THREAD

**Issue by samikeijonen** [WordPress/better-code-editing#89]
In ?-help-button (Additional CSS section) there is `aria-expanded="false"`. But it doesn't change to `aria-expanded="true"` when opened and back to `aria-expanded="false"` when closed.

**Comment by samikeijonen** [comment]
Also when help section is opened if the CSS is empty, value should be `aria-expanded="true"` at that point.

**Comment by westonruter** [comment]
@samikeijonen I think you may be looking at the wrong element. Look at `.customize-section-description` not `.customize-pane-child`.

Upon load Customizer:

![image](https://user-images.githubusercontent.com/134745/30254202-b8cc8d2e-9649-11e7-8e78-a1b2f5621562.png)

Upon clicking Close:

![image](https://user-images.githubusercontent.com/134745/30254216-ef63ac28-9649-11e7-87fb-c8c7faba8f0f.png)

Whether or not `.customize-pane-child` should be getting the same toggle of `aria-expanded` would be an issue for core.

**Comment by samikeijonen** [comment]
Ah ok, I try to open trac ticket for `.customize-pane-child` `aria-expanded` bug.

**Comment by afercia** [comment]
Haven't checked live, just looking at the screenshot above, seems to me `aria-expanded` on a not-focusable div won't do anything useful. The toggle button needs to reflect the expandable panel state it relates to.

**Comment by samikeijonen** [comment]
@westonruter Can you re-open this? When we checked ?-button in the Menu section for example, that button has correct `aria-expanded="false"` and `aria-expanded="true"` when toggling.

**Comment by westonruter** [comment]
@samikeijonen please open a new defect Trac ticket. I just merged this plugin into trunk.

**Comment by afercia** [comment]
@samikeijonen I'm working on a core patch which includes the aria-expanded fix, will update this comment adding the Trac ticket number when I'm done.

See https://core.trac.wordpress.org/ticket/41872

**Comment by samikeijonen** [comment]
Nice.
